subcommands/el2g/gateway.go

```diff
@@ -26,17 +28,19 @@ func init() {
 	}
 	cmd.AddCommand(configCmd)
 	configCmd.Flags().StringVarP(&pkiDir, "pki-dir", "", "", "Directory container factory PKI keys")
+	configCmd.Flags().StringVarP(&hsmModule, "hsm-module", "", "", "Load a root CA key on a PKCS#11 compatible HSM using this module")
+	configCmd.Flags().StringVarP(&hsmPin, "hsm-pin", "", "", "The PKCS#11 PIN to log into the HSM")
+	configCmd.Flags().StringVarP(&hsmTokenLabel, "hsm-token-label", "", "", "The label of the HSM token containing the root CA key")
 }
 
 func doDeviceGateway(cmd *cobra.Command, args []string) {
 	factory := viper.GetString("factory")
 
-	path := filepath.Join(pkiDir, "factory_ca.key")
-	_, err := os.Stat(path)
-	subcommands.DieNotNil(err)
-	path = filepath.Join(pkiDir, "sign_ca_csr")
-	_, err = os.Stat(path)
+	subcommands.DieNotNil(os.Chdir(pkiDir))
```

Comment:
If we use `chdir`, we should change `pki-dir` default value to "." instead of "". Or enforce the use of `--pki-dir`

Reply:
fixed in force-push